JungleBus/Configuration/ReceiveConfigurationExtensions.cs

```diff
@@ -137,6 +137,33 @@ public static IConfigureEventReceiving SetNumberOfPollingInstances(this IConfigu
         }
 
         /// <summary>
+        /// Configure the number of times to attempt to process a message
+        /// </summary>
+        /// <param name="configuration">Configuration to modify</param>
+        /// <param name="count">Number of times to attempt to process a message</param>
+        /// <returns>Modified configuration</returns>
+        public static IConfigureEventReceiving SetMessageRetryCount(this IConfigureEventReceiving configuration, int count)
+        {
+            if (count < 1 || count > 1000)
+            {
+                throw new JungleBusConfigurationException("messageRetryCount", "Number of retries must be between 1 and 1000");

```

Comment:
Another alternative is to defer the creation of the SQS queue to CreateStartableBus in GeneralConfigurationExtensions.